e2e-tests/commands/insurance/declarations/complete-modern-slavery-form.js

```diff
@@ -42,7 +39,9 @@ const completeModernSlaveryForm = ({
     cy.clickNoRadioInput(2);
   }
 
-  cy.completeModernSlaveryFormConditionalFields(conditionalFields);
+  if (submitConditionalFields) {
```

Comment:
will an empty object not enter this if statement?

Reply:
I don't understand the question, can you ellaborate please?

Reply:
ignore me, I can't read